layouts/partials/footer.html

```diff
@@ -0,0 +1,68 @@
+<section class="footer">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-6 col-md-12">
+        <div class="footer-logo">
+          <img src="{{ .Site.Params.footerlogo | absURL}}" alt="logo">
+        </div>
+        <p class="footer-description">
+          {{ .Site.Params.footerDescription}}
+        </p>
+        <div class="footer-text-block">
+          <p>
+            {{ .Site.Params.address.address | safeHTML}}
+          </p>
+        </div>
+      </div>
+      {{ $socialIcon := .Site.Params.social}}
+      <div class="col-lg-6 col-md-12 align-self-end">
+        <div class="footer-icon">
+          <ul>
+            {{ range $socialIcon }}
+						<li >
+              <a target="_blank" href="{{ .url }}">
+                <i
+									class="{{ .icon }}">
```

Comment:
This isn't very accessible for screen reader users, suggest adding `aria-label` to the a tag element.